### ISSUE TITLE
Confirmation modal

### ISSUE THREAD

**Issue by bonucci** [realrashid/sweet-alert#12]
Confirmation box is it ready? For example in case of a delete record? 

Following your example:
```
  public function delete($id)
    {

        alert()->flash('Are you sure?', 'warning',[
            'text' => 'You won\'t be able to revert this!',
            'showCancelButton' => true,
            'confirmButtonColor' => '#3085d6',
            'cancelButtonColor' => '#d33',
            'confirmButtonText' => 'Yes, delete it!',
            // if user clicked Yes, delete it!
            // then this will run
            'deleted' => 'Deleted!',
            'msg' => 'Your file has been deleted.',
            'type' => 'success'
        ]);

```
Is not fired since it gives a blank page. Only works wheren i have a return after the flash message, but it doesnt make sense, since i need the confirmation be fired before is submitted the form.

**Comment by realrashid** [comment]
Are you using v1.0 of this package?
I got what you want but its not implemented in v1.0 or v1.1 it will be added next update this package.

**Comment by heihachi88** [comment]
When do you plan to make it? That would be really good.
